migrate-to-github/README.md

```diff
@@ -0,0 +1,77 @@
+Check the [what-to-migrate.md]() file first, please.
+
+### How we want to do it
+
+We will use the XXX tool (not released yet).
+  - still thinking about its name :D I think about two possibilities:
+    - `volagit` or `forgit` -> these are just random names containing `git`... I think
+       about them because they sounds cool :D
+
+### What tools it will use
+
+- [Ogr](https://github.com/packit/ogr)
+  - A tool which wraps Pagure, Gitlab and Github API into one API
+  - There's 95% of API calls we need to use 
+- [GitPython](https://gitpython.readthedocs.io/en/stable/)
+  - A tool which wraps git cli to a Python api
+  - only when Github, Gitlab API doesn't allow something (e.g. `git reset`). In other cases Ogr
+    includes all the stuff we need.
+- That's it
+  
+### What we need to use to achieve the goals in [what-to-migrate.md]()
+
+- Create a bot account which will transfer the PRs/Issues to Github
+- Create Github mirror of Copr project
+  - Copr migrated from Github to Pagure, where it had already some 30 PRs. The project
+    on Pagure did not migrate these PRs and started on PR id 2 -> the previous 30 PRs
+    are lost from context. But in the current Github mirror these PRs are there and
+    we need to create a new clean mirror to match all the Pagure IDs.
+- Give this bot rights to write to Copr project
+- Create Github API token for this bot account
+  - Security: Give the token minimal amount of privileges -> it won't need to delete
+    something so don't enable this! (but on the other hand if something will be
+    screwed up - it's still just a new mirror repo so no big deal :D) And delete
+    the token once migration is done.
+  - The API token needs to read and write, check these options when creating API 
+    (I am still checking if we really needs all of these):
+  - https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes
+    - public_repo
+    - read:project
+    - read:user (for migrating assignees)
+- Create Pagure API token
+  - Security: the same reason as above
+  - The API token needs only to read the project, nothing else.
+  - Documentation missing???:
+    - ???
+- Create a config for the script
+- Run the script
+- Have a cup of coffee and relax
```

Comment:
If I understand correctly, this whole process will be done to a fork to make sure that the script works as it should? 
And then, once considered safe, we will run this against the actual
https://github.com/fedora-copr/copr

Reply:
Yes. I don't want to run it on actual Copr's GH mirror on the first try.